topics/proteomics/tutorials/clinical-mp-data-verification/tutorial.md

```diff
@@ -0,0 +1,406 @@
+
+
+> <hands-on-title> Collasping PSM rank files into a singular dataset using Collapse Collection </hands-on-title>
+>
+> 1. {% tool [Collapse Collection](toolshed.g2.bx.psu.edu/repos/nml/collapse_collections/collapse_dataset/5.1.0) %} with the following parameters:
+>    - {% icon param-file %} *"Collection of files to collapse into single dataset"*: `psm_rank_txt` (output of **PepQuery2** {% icon tool %})
+>    - *"Keep one header line"*: `Yes`
+>
+>
+{: .hands_on}
+
+
+
+## Filtering out confident peptides from PepQuery2 with **Filter**
+
+Now, we want to filter for confident peptides from PepQuery2 and prepare them for the Query Tabular tool.
+
+> <hands-on-title> Filter </hands-on-title>
+>
+> 1. {% tool [Filter](Filter1) %} with the following parameters:
+>    - {% icon param-file %} *"Filter"*: `output` (output of **Collapse Collection** {% icon tool %})
+>    - *"With following condition"*: `c20=='Yes'`
+>    - *"Number of header lines to skip"*: `1`
+>
+>
+{: .hands_on}
+
+
+> <hands-on-title> Remove header line from filtered PepQuery peptides with Remove beginning </hands-on-title>
+>
+> 1. {% tool [Remove beginning](Remove beginning1) %} with the following parameters:
+>    - {% icon param-file %} *"from"*: `out_file1` (output of **Filter** {% icon tool %})
+>
+>
+{: .hands_on}
+
+
+> <hands-on-title> Cut (select out) peptide sequences from PepQuery output with Cut </hands-on-title>
+>
+> 1. {% tool [Cut](Cut1) %} with the following parameters:
+>    - *"Cut columns"*: `c1`
+>    - {% icon param-file %} *"From"*: `out_file1` (output of **Remove beginning** {% icon tool %})
+>
+>
+{: .hands_on}
+
+
+##  **Query Tabular**
+
+We will use the Query Tabular tool { %cite Johnson2019 % } to search the PepQuery-verified peptides against the concatenated dataset that contains peptides and proteins from SearchGUI/Peptide and MaxQuant.
+
+> <hands-on-title> Querying verified peptides </hands-on-title>
+>
+> 1. {% tool [Query Tabular](toolshed.g2.bx.psu.edu/repos/iuc/query_tabular/query_tabular/3.3.0) %} with the following parameters:
+>    - In *"Database Table"*:
+>        - {% icon param-repeat %} *"Insert Database Table"*
+>            - {% icon param-file %} *"Tabular Dataset for Table"*: `out_file1` (output of **Cut** {% icon tool %})
+>            - In *"Table Options"*:
+>                - *"Specify Name for Table"*: `pep`
+>                - *"Specify Column Names (comma-separated list)"*: `mpep`
+>        - {% icon param-repeat %} *"Insert Database Table"*
+>            - {% icon param-file %} *"Tabular Dataset for Table"*: `out_file1` (output of **Concatenate datasets** {% icon tool %})
+>            - In *"Table Options"*:
+>                - *"Specify Name for Table"*: `prot`
+>                - *"Specify Column Names (comma-separated list)"*: `pep,prot`
+>    - *"SQL Query to generate tabular output"*: `select pep.mpep, prot.prot
+>    `FROM pep
+>    INNER JOIN  prot on pep.mpep=prot.pep `
```

Comment:
maybe a little comment box here explaining what the query does on a conceptual level, for those not familiar with SQL?